### PR TITLE
Dynamically stylable SVG

### PR DESCRIPTION
**Pull request by OiYouYeahYou**
In response to @niik's wonderful comment here https://github.com/desktop/desktop/issues/1515#issuecomment-336532891, I have made the SVGs dynamically stylable.

Currently, they have inline styles allowing them to be used as images with no changes in .scss or .tsx

If you accept these changes I'll follow up with relevant another PR with relevant .tsx changes and .scss additions to fully utilise

### CODE REVIEW

File: app/static/common/empty-no-file-selected.svg [hunk 1]
```diff
@@ -1 +1,29 @@
...
+    </title>
+    <style type="text/css">
+        <![CDATA[
+            .outline1 {
+                fill: #cadef3;
+            }
+            #paperShadow {
+                fill: #ebf2fa;
+            }
+            #paperBackround {
```
Comment (niik) [inline app/static/common/empty-no-file-selected.svg]: Minor nit, `backround` is missing a `g`

File: app/static/common/empty-no-commit.svg [hunk 2]
```diff
@@ -1 +1,24 @@
...
+    <title>
+        No commit selected
+    </title>
+    <style type="text/css">
+        <![CDATA[
+            .outline1 {
+                fill: #cadef3;
+            }
+            .shadow {
+                fill: #ebf2fa;
```
Comment (niik) [inline app/static/common/empty-no-commit.svg]: I think we can just use the default color here and drop the opacity of the shadow to 50%

File: app/static/common/empty-no-file-selected.svg (hunk 1 above)
Comment (niik) [reply app/static/common/empty-no-file-selected.svg]: This was an early comment. Ignore this as we'll be getting rid of it completely

**Comment by OiYouYeahYou** [comment]
Sorry to be difficult. But I deleted the repo as I, naively, did this on my master branch. 

Is it ok to make a new PR? As I can't even edit this PR. That being said would you like the PR to be too a 'non-master' branch?

**Comment by niik** [comment]
@OiYouYeahYou You're not being difficult at all, quite the opposite 😄 

Just open a new PR and target master. We'll close this one out and reference this discussion from the new one.

**Comment by OiYouYeahYou** [comment]
Done: https://github.com/desktop/desktop/pull/3773

**Pull request closed without merging**